.travis.yml

```diff
@@ -6,26 +6,20 @@ rvm:
   - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
+  - 2.4.0
   - rbx-2
   - jruby
```

Comment:
I think this is for JRuby 1.x, which maybe we still support, same for RBX..
Neither of those are failing, so I don't really care too much.